New-Win32Package.ps1

```diff
@@ -347,10 +348,18 @@ process {
                 }
 
                 #region Create the intunewin package
+                # Adjust params for New-IntuneWin32AppPackage using PSAppDeployToolkit
+                $IntuneWinSetupFile = $Manifest.PackageInformation.SetupFile
+                if (Test-Path -Path $([System.IO.Path]::Combine($AppPath, "Source", "Deploy-Application.ps1"))) {
+                    # Revert source path
+                    $SourcePath = [System.IO.Path]::Combine($WorkingPath, $ApplicationName, "Source")
+                    $IntuneWinSetupFile = "Deploy-Application.exe"
```

Comment:
The `SetupFile` itself must exist or the process will be interrupted by the https://github.com/MSEndpointMgr/IntuneWin32App/blob/master/Public/New-IntuneWin32AppPackage.ps1#L68 process.
To avoid this, I have tried to specify the `SetuFile` statically.